mne/preprocessing/tests/test_realign.py

```diff
@@ -45,45 +52,87 @@ def test_realign(ratio_other, start_raw, start_other, stop_raw, stop_other):
     data_raw = np.array(
         [
             interp1d(times, d, kind)(raw_times)
-            for d, kind in ((signal, "linear"), (stim, "nearest"))
+            for d, kind in (
+                (stim, "nearest"),
+                (sig_hann, "linear"),
+                (sig_box, "linear"),
```

Comment:
boxcar interpolation should use `"nearest"` (if there's reason to keep `sig_box` in addition to `stim`)